api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimService.java

```diff
@@ -41,6 +41,28 @@ public Claimant createClaim(Claim claim) {
         }
     }
 
+    private EligibilityStatus determineEligibilityStatusForHousehold(EligibilityResponse eligibilityResponse) {
+        return eligibleClaimExistsForHousehold(eligibilityResponse)
+                ? EligibilityStatus.DUPLICATE
+                : eligibilityResponse.getEligibilityStatus();
+    }
+
+    private boolean eligibleClaimExistsForHousehold(EligibilityResponse eligibilityResponse) {
+        String dwpHouseholdIdentifier = eligibilityResponse.getDwpHouseholdIdentifier();
+
+        if (dwpHouseholdIdentifier != null) {
+            return claimantRepository.eligibleClaimExistsForDwpHousehold(dwpHouseholdIdentifier);
+        }
+
+        String hmrcHouseholdIdentifier = eligibilityResponse.getHmrcHouseholdIdentifier();
+
+        if (hmrcHouseholdIdentifier != null) {
+            return claimantRepository.eligibleClaimExistsForHmrcHousehold(hmrcHouseholdIdentifier);
+        }
+
+        return false;
+    }
```

Comment:
There is a chance (e.g. if a household moves from CTC to UC within the past cycle) that we get both household identifiers back. At that point we should check against both identifiers:
```
    private boolean eligibleClaimExistsForHousehold(EligibilityResponse eligibilityResponse) {
        String dwpHouseholdIdentifier = eligibilityResponse.getDwpHouseholdIdentifier();
        boolean dwpClaimExists = dwpHouseholdIdentifier != null &&  claimantRepository.eligibleClaimExistsForDwpHousehold(dwpHouseholdIdentifier);

         String hmrcHouseholdIdentifier = eligibilityResponse.getHmrcHouseholdIdentifier();
         boolean hmrcClaimExists = hmrcHouseholdIdentifier != null && claimantRepository.eligibleClaimExistsForHmrcHousehold(hmrcHouseholdIdentifier);

         return dwpClaimExists || hmrcClaimExists;
    }
```